src/signalingclient/SignalingClient.ts

```diff
@@ -38,29 +38,29 @@ export default interface SignalingClient {
    * is open, then that is first closed (and the WebSocketClosed event is sent out) before the
    * open is tried.
    *
-   * @param {SignalingClientConnectionRequest} request The request to enqueue.
+   * @param request The request to enqueue.
    */
   openConnection(request: SignalingClientConnectionRequest): void;
 
   /**
    * Sends a ping or pong with an id.
    *
-   * @param {SdkPingPongFrame} the ping or pong frame to send.
-   * @return number the ms timestamp when the message was sent.
+   * @param pingPongFrame ping or pong frame to send.
+   * @returns the ms timestamp when the message was sent.
    */
   pingPong(pingPongFrame: SdkPingPongFrame): number;
 
   /**
    * Sends a join frame with the given settings.
    *
-   * @param {SignalingClientJoin} settings How to configure the Join frame.
+   * @param settings How to configure the Join frame.
    */
   join(settings: SignalingClientJoin): void;
 
   /**
    * Sends a subscribe frame with the given settings.
    *
-   * @param {SignalingClientSubscribe} settings How to configure the Subscribe frame.
+   * @param settings How to configure the Subscribe frame.
```

Comment:
So, does it still display the parameter type name or its fine as we already show in the method signature. Does it still highlight or linkify if users want to click and go to it?

We are removing in other classes as well so just trying to understand the end output here.

Reply:
The syntax I'm fixing here is JSDoc. JavaScript doesn't have a way to type-annotate parameters, so it uses this `{syntax}` to indicate the type.

TypeScript has type annotations, so TypeDoc and other tools _already know_ that `settings` is a `SignalingClientSubscribe` instance, and can link directly to the correct type.

The output is exactly the same; TypeDoc is ignoring the `{SignalingClientSubscribe}` here.

Reply:
Here's what it looks like.

<img width="427" alt="Screen Shot 2021-04-08 at 2 45 46 PM" src="https://user-images.githubusercontent.com/40039049/114100421-1bcfcc80-9879-11eb-9163-377a6b57529c.png">

Reply:
Cool, thanks for the explanation!